marlowe-runtime/src/Language/Marlowe/Runtime/Core/Api.hs

```diff
@@ -1,3 +1,5 @@
+-- editorconfig-checker-disable-file
+
```

Comment:
can remove

Reply:
Fixed in 34f1d7e5d89e68996254f165386a3264372c3252.